docs/cody/model-configuration/index.mdx

```diff
@@ -0,0 +1,553 @@
+
+## Quickstart
+
+The quickest and recommended way to set up model configuration is by using Sourcegraph-supplied models through the Cody Gateway.
+
+For a minimal configuration example, see [Configure Sourcegraph-supplied models](/cody/model-configuration#configure-sourcegraph-supplied-models).
+
+## Sourcegraph-supplied models
+
+Sourcegraph-supplied models, accessible through the [Cody Gateway](/cody/core-concepts/cody-gateway), are managed via your site configuration.
+
+For most administrators, relying on these models alone ensures access to high-quality models without needing to manage specific configurations.
+
+Usage of these models is controlled through the `"modelConfiguration.sourcegraph"` field in the site config.
+
+### Disable Sourcegraph-supplied models
+
+To disable all Sourcegraph-supplied models and use only the models explicitly defined in your site configuration, set the `"sourcegraph"` field to `null` (see example below).
+
+```json
+"cody.enabled": true,
+"modelConfiguration": {
+  "sourcegraph": null, // ignore Sourcegraph-supplied models
+  "providerOverrides": {
+    // define access to the LLM providers
+  },
+  "modelOverrides": {
+    // define models available via providers defined in the providerOverrides
+  },
+  "defaultModels": {
+    // set default models per Cody feature from the list of models defined in modelOverrides
+  }
+}
+```
+
+### Configure Sourcegraph-supplied models
+
+The minimal configuration for Sourcegraph-supplied models is:
+
+```json
+"cody.enabled": true,
+"modelConfiguration": {
+  "sourcegraph": {}
+}
+```
+
+The above configuration sets up the following:
+
+-   Sourcegraph-supplied models are enabled (`sourcegraph` is not set to `null`).
+-   Requests to LLM providers are routed through the Cody Gateway (no `providerOverrides` field specified).
+-   Sourcegraph-defined default models are used for Cody features (no `defaultModels` field specified).
+
+There are three main settings for configuring Sourcegraph-supplied LLM models:
+
+| Field                                                     | Description                                                                                            |
+| --------------------------------------------------------- | ------------------------------------------------------------------------------------------------------ |
+| `endpoint`                                                | (Optional) The URL for connecting to Cody Gateway, defaulting to the production instance.              |
+| `accessToken`                                             | (Optional) The access token for connecting to Cody Gateway, which defaults to the current license key. |
+| [`modelFilters`](/cody/model-configuration#model-filters) | (Optional) Filters specifying which models to include from Cody Gateway.                               |
+
+### Model Filters
```

Comment:
Nit: we use Sentence case for our headings. In this PR, they are a mix of Sentence case and Title Case. We should probably use Sentence case everywhere.

```suggestion
### Model filters
```